IdentityCore/src/cache/mac/MSIDMacACLKeychainAccessor.h

```diff
@@ -34,6 +34,7 @@ NS_ASSUME_NONNULL_BEGIN
 
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->